src/PinchZoomPan.js

```diff
@@ -552,22 +555,23 @@ export default class PinchZoomPan extends React.Component {
     //lifecycle methods
     render() {
         const childElement = React.Children.only(this.props.children);
-        const { zoomButtons, maxScale, debug } = this.props;
+        const { zoomButtons, maxScale, debug, containerStyle } = this.props;
         const { scale } = this.state;
 
         const touchAction = this.controlOverscrollViaCss
             ? browserPanActions(this.state) || 'none'
             : undefined;
 
-        const containerStyle = {
+        const style = {
             width: '100%', 
             height: '100%',
             overflow: 'hidden',
             touchAction: touchAction,
```

Comment:
Will thsi override the above styels or ?

Reply:
Yes, will override the above styles.